src/nodes.coffee

```diff
@@ -4525,3 +4627,20 @@ mergeAstLocationData = (nodeA, nodeB) ->
     ]
     start: lesser  nodeA.start, nodeB.start
     end:   greater nodeA.end,   nodeB.end
+
+# Convert Jison-style node class location data to Babel-style location data
+locationDataToAst = ({first_line, first_column, last_line, last_column, range}) ->
```

Comment:
Can we rename this to something like `jisonLocationDataToAstLocationData` to be clearer?

Reply:
Sure updated

That name is perhaps a little long, would any similar alternatives be good enough in your opinion? Eg `jisonToAstLocationData` or `jisonLocationDataToAst`?

Reply:
What I didn’t like about ones that ended in `ToAst` was that it implied that it was perhaps returning more than just the location data. Let’s leave it long, I want to discourage this function’s use anyway 😉because it’s only needed when we’re not following the pattern of one node class per one AST node.